public/worker.js

```diff
@@ -5,6 +5,8 @@ self.q = self.Q
 
 addEventListener('message', msg => {
   if (msg.data && msg.data.action == 'generate') {
+    console.log(`${JSON.stringify(msg.data.twoSourcesChance)}`)
+    console.log(`${JSON.stringify(msg.data.wallChance)}`)
```

Comment:
Minor nitpick, extra debug `console.log` calls left in

Reply:
should be removed